.gitignore

```diff
@@ -4,7 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
-/packages/*/node_modules
+/packages/*/
```

Comment:
why do we need to ignore all file in packages ?

Reply:
I think it should be `/packages/*/tests`

Reply:
Sorry its a mistake from me while resolving conflicts, no need to change anything here